modules/logging_configs/main.tf

```diff
@@ -6,7 +6,7 @@ locals {
   fire_lens_container = <<EOF
   {
     "essential": true,
-    "image": "906394416424.dkr.ecr.${var.region}.amazonaws.com/aws-for-fluent-bit:stable",
+    "image": "514496004130.dkr.ecr.${var.region}.amazonaws.com/fluentbit-custom-eng:1.0.0",
```

Comment:
We should be fine pulling this image with the current permissions https://github.com/smartrent/nerves_hub_infrastructure/blob/main/all_environments/iam.tf#L20
![Screen Shot 2023-01-23 at 12 37 22 PM](https://user-images.githubusercontent.com/54445269/214133262-bd84d0da-16b5-4d4f-ab68-fc504379f332.png)

Reply:
sigh, yeah this should be fine. Ideally, this permission is scoped down to just the repos that this service needs. otherwise, anyone who can push a change to this ecs service can pull any docker image in the aws ENG account.